### PR TITLE
chore: remove unused functions

### PR DESCRIPTION
**Pull request by jonaslagoni**
**Description**
This PR removes some unused functions.

### CODE REVIEW

File: src/helpers/TypeHelpers.ts [hunk 1]
```diff
@@ -1,29 +1,6 @@
 import { CommonModel } from '../models';
```
Comment (Samridhi-98) [inline src/helpers/TypeHelpers.ts:1]: All the unused import statements can be removed 😊

**Review by Samridhi-98 — CHANGES_REQUESTED** [review]

File: src/helpers/TypeHelpers.ts (hunk 1 above)
Comment (magicmatatjahu) [reply src/helpers/TypeHelpers.ts:1]: Yeah, you have even lint error with that in testing workflow.

Comment (jonaslagoni) [inline src/helpers/TypeHelpers.ts:1]:
```suggestion
```

**Pull request merged**